bin/proving-service/src/commands/worker.rs

```diff
@@ -15,6 +48,9 @@ pub struct StartWorker {
     /// The port of the worker
     #[clap(short, long, default_value = "50051")]
     port: u16,
+    /// The type of prover that the worker will be
+    #[clap(flatten)]
+    prover_type: ProverTypeSupport,
```

Comment:
Does this mean that a worker can only prove one type of proofs after instantiation? If so, I wonder if we need this limitation.

Reply:
No, the worker can support multiple types during it's uptime. `ProverTypeSupport` just holds the supported proof types during initialization of the worker.